packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -77,72 +74,81 @@ export interface FormFieldProps {
   readonly inline?: boolean;
 
   /**
-   * Highlights the field red to indicate an error.
-   */
-  readonly invalid?: boolean;
-
-  /**
-   * Specifies the maximum numerical or date value that a user can type
+   * Show a spinner to indicate loading
    */
-  readonly max?: number;
+  loading?: boolean;
 
   /**
-   * Maximum character length for an input. This also changes the width to
-   * roughly the same size as the max length. This is to communicate that the
-   * user that on certain cases, they can only type a limited amount of
-   * characters.
+   * Name of the input.
    */
-  readonly maxLength?: number;
+  readonly name?: string;
 
   /**
-   * Specifies the minimum numerical or date value that a user can type
+   * Simplified onChange handler that only provides the new value.
+   * @param newValue
    */
-  readonly min?: number;
+  onChange?(newValue: string | number | boolean | CivilTime): void;
 
   /**
-   * Name of the input.
+   * Callback to get the the status and message when validating a field
+   * @param message
    */
-  readonly name?: string;
+  onValidation?(message: string): void;
 
   /**
    * Hint text that goes above the value once the form is filled out.
    */
   readonly placeholder?: string;
 
   /**
-   * Prevents users from editing the value.
+   * Adjusts the interface to either have small or large spacing.
    */
-  readonly readonly?: boolean;
+  readonly size?: "small" | "large";
 
   /**
-   * Exclusively for textareas. Specifies the visible height of a textarea.
+   * Set the component to the given value.
    */
-  readonly rows?: number;
+  readonly value?: string | number | CivilTime;
```

Comment:
Again, shouldn't be adding in types `CivilTime` should be specific to `InputTime` i believe

Reply:
Fixed: ac09001